proto/shentu/bounty/v1/tx.proto

```diff
@@ -15,6 +15,9 @@ option go_package = "github.com/shentufoundation/shentu/x/bounty/types";
 service Msg {
   // CreateProgram defines a method for creating a new program.
   rpc CreateProgram(MsgCreateProgram) returns (MsgCreateProgramResponse);
+
+  // SubmitFinding defines a method for submitting a new finding.
+  rpc SubmitFinding(MsgSubmitFinding) returns (MsgSubmitFindingResponse);
 }
 
 // MsgCreateProgram defines a SDK message for creating a new validator.
```

Comment:
Wrong comment `creating a new validator` (not introduced by this PR)

Reply:
fixed